### PR TITLE
Realtor report

### PR DESCRIPTION
**Pull request by khaosans**

### CODE REVIEW

**Comment by lby221** [comment]
Seems good to me.

**Comment by AaronHampshire** [comment]
Not good. Don't merge.

**Comment by lby221** [comment]
Won't merge. 

Baiyu Liu

Thursday, July 16, 2015, 7:18 PM -0700 from Aaron Hampshire  notifications@github.com:

File: RealEstate/apps/core/templates/core/realtorHome.html [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+
+            <!-- Modal content-->
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                </div>
+                <div class="modal-body">
+                    {% block content %}
+                        {% include 'core/base.html' %}
+                    {% endblock content %}
```
Comment (AaronHampshire) [inline RealEstate/apps/core/templates/core/realtorHome.html]: Modal don't need entire base html here. Just form. This puts header bar and everything.

File: RealEstate/apps/core/templates/core/realtorHome.html [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+				<div class="col-sm-6 vcenter">
+				    {% for homebuyer in homebuyers %}
+				        <h4>{{homebuyer}}</h4>
+	                {% endfor %}
+				</div>
+				<div class="col-sm-4 vcenter">
+				    Right Holding Block
+				</div>
+				<div class="col-sm-2 vcenter">
+					<a href="homebuyerreport">Report</a>
```
Comment (AaronHampshire) [inline RealEstate/apps/core/templates/core/realtorHome.html:29]: Look at other examples of how to link to urls contained in urls.py. This isn't quite correct.

File: RealEstate/urls.py [hunk 3]
```diff
@@ -29,6 +29,7 @@
         'django.contrib.auth.views.logout_then_login', name='auth_logout'),
     url(r'^change-password/$', 'django.contrib.auth.views.password_change',
         {'post_change_redirect': 'home'}, name='password_change'),
+    url(r'^homebuyerreport/$',PendingViews.HomebuyerReportView.as_view(),name='report'),
```
Comment (AaronHampshire) [inline RealEstate/urls.py:32]: There will be a homebuyer report for each couple. Take a look at how this is done for house evals. Needs more than just static endpoint. The name 'report' will be used in your link above. Again, follow the eval model.

File: RealEstate/urls.py [hunk 4]
```diff
@@ -29,6 +29,7 @@
...
```
Comment (AaronHampshire) [inline RealEstate/urls.py:33]: Let's get rid of blank line for cleanliness.

File: RealEstate/apps/core/templates/core/realtorHome.html [hunk 5]
```diff
@@ -1,5 +1,56 @@
...
+				</div>
+			</div>
+		</li>
+		{% endfor %}
+
+        <button type="button" class="btn btn-primary vcenter" data-toggle="modal" data-target="#AddCoupleModal">Add New Couple</button>
```
Comment (AaronHampshire) [inline RealEstate/apps/core/templates/core/realtorHome.html]: Let's put some padding on this button and put in `<center>` blocks to keep things uniform.

File: RealEstate/apps/core/templates/core/realtorHome.html [hunk 6]
```diff
@@ -1,5 +1,56 @@
...
+        {% for couple, homebuyers in couples %}
+        <li class="list-group-item">
+			<div class="row">
+				<div class="col-sm-6 vcenter">
+				    {% for homebuyer in homebuyers %}
+				        <h4>{{homebuyer}}</h4>
+	                {% endfor %}
+				</div>
+				<div class="col-sm-4 vcenter">
+				    Right Holding Block
```
Comment (AaronHampshire) [inline RealEstate/apps/core/templates/core/realtorHome.html:26]: This line was meant to be a help to you guys. It wasn't meant to be merged into our main dev branch.

File: RealEstate/apps/core/templates/core/realtorHome.html [hunk 7]
```diff
@@ -1,5 +1,56 @@
...
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+</style>
 {% endblock %}
+
+{% block body %}
+    <h2><strong> Dashboard </strong> </h2>
+    <ul class="list-group">
```
Comment (AaronHampshire) [inline RealEstate/apps/core/templates/core/realtorHome.html:16]: Let's also provide some space between the header and list-group. Something around 60px. To do that inline for the header: style="margin-bottom : 60px;"

**Comment by AaronHampshire** [comment]
So this is the tough thing about trying to push a small feature into dev before the actual larger feature is established. You get to deal with all the nitty-gritty style first. 

My gut says, you should probably hold off on this and wait until realtor-dashboard has been merged. Then you won't need to worry about a lot of this.

**Comment by khaosans** [comment]
no thats fine stephen just wanted me to put in a pr

**Pull request closed without merging**